cmd/tokens.go

```diff
@@ -0,0 +1,124 @@
+package cmd
+
+import (
+	"errors"
+)
+
+var errMissingToken = errors.New("missing token")
+
+func init() {
+	register(&tokensCmd{})
+}
+
+type tokensCmd struct {
+	Create  createTokensCmd  `command:"create" description:"Create a new access token"`
+	List    lsTokensCmd      `command:"ls" description:"List available access tokens"`
+	Compare compareTokensCmd `command:"compare" description:"Check if access token is valid"`
+	Remove  rmTokensCmd      `command:"rm" description:"Remove a specific access token"`
+}
+
+func (x *tokensCmd) Name() string {
+	return "tokens"
+}
+
+func (x *tokensCmd) Short() string {
+	return "Manage Cafe developer access tokens"
+}
+
+func (x *tokensCmd) Long() string {
+	return `
+Tokens allow other peers to register with a Cafe peer.
+Use this command to create, list, compare, and remove tokens required for access to this peer's Cafe.
+`
+}
+
+type createTokensCmd struct {
+	Client ClientOptions `group:"Client Options"`
+}
+
+func (x *createTokensCmd) Usage() string {
+	return `
+
+Generates an access token (32 random bytes) and saves bcrypt encrypted version for future lookup.
+The response contains a base58 encoded version of the random bytes token.
```

Comment:
The token is actually a `ksuid() + "+" + base58(randomBytes)` string. Could be very explicit here, though code documents this fairly well.

Reply:
as per offline conversation, we'll change this to just use random bytes for id and avoid additional +